requirements.txt

```diff
@@ -1,5 +1,6 @@
 --index-url https://pypi.python.org/simple/
 axisregistry==0.4.2
+babelfont==3.0.1
```

Comment:
I'd prefer it if we didn't use babelfont for this check. It isn't too difficult to do the check using just a TTFont object.

Reply:
I've rebased this PR and fixed conflicts again. It looks good, so I'll merge it as-is (still using babelfont) and may later replace it by inline routines directly using TTFont. I just don't see any actual harm in using babelfont, though, if it makes it easier to quickly draft new checks, like @yanone did.